regparser/grammar/unified.py

```diff
@@ -72,6 +74,22 @@ def appendix_section(match):
                     | atomic.upper_p)
        ).setParseAction(appendix_section).setResultsName("appendix_section"))
 
+# "the" appendix implies there's only one, so it better be appendix A
+section_of_appendix_to_this_part = (
+    atomic.section_marker
+    + atomic.upper_roman_a.setResultsName("appendix_section")
+    + Literal("of the appendix to this part").setResultsName("appendix").setParseAction(lambda: 'A')
```

Comment:
Do you think the "to this part" is needed? Also might be worth reusing the `of` connective, though I don't remember which file that's in.

Setting it to `A` is a lil sneaky, but I think safe. Mind adding a comment to explain why?